docs/refguide/observer-component.md

```diff
@@ -164,7 +164,6 @@ import {observer} from "mobx-react";
 
 * `componentWillReact` doesn't take arguments
 * `componentWillReact` won't fire before the initial render (use `componentWillMount` instead)
-* `componentWillReact` won't fire when receiving new props or after `setState` calls (use `componentWillUpdate` instead)
```

Comment:
@mweststrate well this PR is about removing it. I propose to leave it there, but just making it true by removing the negative.

It is my understanding that for mobx-react 4.0.0 and up this must have changed since the `state` and `props` is now observable as well.